docs/guides/src/main/server/configuration.adoc

```diff
@@ -118,25 +112,25 @@ hostname mykeycloak.acme.com
 .Start the server
 <@kc.start/>
 
-By following that approach, the server is going to skip some steps at startup and start faster.
+By using the `keycloak.conf` file, the server can omit some steps at startup. As a result, the server starts faster.
 
-== Using the `auto-build` option when starting the server
+== Two configuration stages from one command
```

Comment:
I'd like to leave the "auto-build" in the headline. This will be the searchterm everybody will search for who really wants to use this, as it is mentioned everywhere form CLI help to blog entries etc.

Suggestion: Leave as is, or something like "Using the `auto-build` option to automatically detect when the server needs a build" (which is too long imo)